src/test/ui/chalkify/fixed-by-chalk/chalk_initial_program.stderr

```diff
@@ -0,0 +1,24 @@
+
+chalk yielded item that legacy mode did not return: FulfillmentError(
+    Obligation {
+        predicate: Binder(WellFormed(S), []),
+        depth: 0,
+    },
+    Unimplemented,
+)
```

Comment:
This is likely due to us not elaborating the environment because we aren't under `-Zchalk`.